src/Scrapers/Pipeline/Types/ScrapeConfig.ts

```
@@ -0,0 +1,111 @@
+/** Configuration for fetching transactions per account. */
+interface ITransactionsFetchConfig<TRaw> {
+  /** HTTP method for transactions endpoint. */
+  readonly method: HttpMethod;
+  /**
+   * Build the URL/params for one account's transactions.
+   * @param accountId - The account/card ID.
+   * @param startDate - Formatted start date string.
+   * @returns URL path + optional POST data.
+   */
+  readonly buildRequest: (
+    accountId: string,
```

ESLint

[error] 'accountId' is defined but never used.

(no-unused-vars)